Cargo.toml

```diff
@@ -51,6 +51,9 @@ arrow-buffer = { version = "41.0.0", default-features = false }
 arrow-schema = { version = "41.0.0", default-features = false }
 arrow-array = { version = "41.0.0", default-features = false, features = ["chrono-tz"] }
 parquet = { version = "41.0.0", features = ["arrow", "async", "object_store"] }
+graphviz-rust = "0.6.2"
```

Comment:
These crates appear to be relatively new and from a single author -- 

https://crates.io/crates/graphviz-rust
https://crates.io/crates/dot-structures
https://crates.io/crates/dot-generator

Could you comment on how much value they add rather than implementing the relevant portions within datafusion (graphviz is not a complicated format and doesn't typically need a library to create, in my experience)

Reply:
@liurenjie1024  -- do we really need new dependencies? I would prefer to avoid this if at all possible